test/Serializer/ReviewedPreprintNormalizerTest.php

```diff
@@ -0,0 +1,129 @@
+        $this->normalizer = new ReviewedPreprintNormalizer(new ReviewedPreprintsClient($this->getHttpClient()));
+        $this->normalizer->setNormalizer($apiSdk->getSerializer());
+        $this->normalizer->setDenormalizer($apiSdk->getSerializer());
+    }
+
+    /**
+     * @test
+     */
+    public function it_is_a_normalizer()
+    {
+        $this->assertInstanceOf(NormalizerInterface::class, $this->normalizer);
+    }
+
+    /**
+     * @test
+     * @dataProvider canNormalizeProvider
+     */
+    public function it_can_normalize_reviewed_preprints($data, $format, bool $expected)
+    {
+        $this->assertSame($expected, $this->normalizer->supportsNormalization($data, $format));
+    }
+
+    public function canNormalizeProvider() : array
+    {
+        $reviewedPreprint = Builder::for(ReviewedPreprint::class)->sample('minimum');
+
+        return [
+            'preprint' => [$reviewedPreprint, null, true],
+            'non-preprint' => [$this, null, false],
+        ];
+    }
+
+    /**
+     * @test
+     */
+    public function it_normalize_reviewed_preprints()
+    {
+        $expected = [
+            'id' => '1',
+            'title' => 'title',
+            'status' => 'reviewed',
+            'stage' => 'published',
+        ];
+
+        $reviewedPreprint = Builder::for(ReviewedPreprint::class)->sample('minimum');
+        $res = $this->normalizer->normalize($reviewedPreprint);
+        $this->assertSame($expected, $res);
+    }
+
+    /**
+     * @test
+     */
+    public function it_is_a_denormalizer()
+    {
+        $this->assertInstanceOf(DenormalizerInterface::class, $this->normalizer);
+    }
+
+    /**
+     * @test
+     * @dataProvider canDenormalizeProvider
+     */
+    public function it_can_denormalize_reviewed_preprints($data, $format, array $context, bool $expected)
+    {
+        $this->assertSame($expected, $this->normalizer->supportsDenormalization($data, $format, $context));
+    }
+
+    public function canDenormalizeProvider() : array
+    {
+        return [
+            'preprint' => [[], ReviewedPreprint::class, [], true],
+            'non-preprint' => [[], get_class($this), [], false],
+        ];
+    }
+
+    /**
+     * @test
+     */
+    public function it_denormalize_reviewed_preprints()
+    {
+        $json = [
+            'id'=> '1',
+            'status' => 'reviewed',
+            'title' => 'title',
+            'stage' => 'published',
+        ];
+        $expected = Builder::for(ReviewedPreprint::class)->sample('minimum');
+
+        $this->assertObjectsAreEqual($expected, $this->normalizer->denormalize($json, ReviewedPreprint::class));
+    }
+
+    protected function class() : string
+    {
+        return ReviewedPreprint::class;
+    }
+
+    protected function samples()
+    {
+        yield __DIR__.'/../../vendor/elife/api/dist/samples/reviewed-preprint/v1/*.json';
+        yield __DIR__.'/../../vendor/elife/api/dist/samples/reviewed-preprint-list/v1/*.json#items';
```

Comment:
We need to remove the indexContent property from the sample listing as it is not expected. This a change to the api-raml.

Will submit a PR.

Reply:
https://github.com/elifesciences/api-raml/pull/285

Reply:
Although tests are failing on ci still the issues have been resolved. I tagged the elife/api 2.7.0 before the dist folder had been regenerated so I had to delete the tag and the recreate it as 2.7.0

I'm trying now to show the tests passing by altering the elife/api dependency to dev-master. Once it passes I will change it back to ^2.7. If still failing then could @scottaubrey or @lsh-0 take a look please.

Reply:
> I'm trying now to show the tests passing by altering the elife/api dependency to dev-master. Once it passes I will change it back to ^2.7. If still failing then could @scottaubrey or @lsh-0 take a look please.

Tests are now passing for the elife/api:^2.7 dependency!!